docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 version: "2.1"
 
 networks:
-  # docker network create gather_internal
+  # docker network create aether_internal
```

Comment:
Id this is Gather, why the network is called `aether_...`? Just a question

Reply:
I think it's because `aether_internal` is the name of the network used by Aether Connect (Producer, Kafka and Consumers). We could reconfigure all those to use `gather_internal` too, but I think it's probably OK name the network after the platform rather than the application.